airflow/providers/elasticsearch/CHANGELOG.rst

```diff
@@ -27,6 +27,17 @@
 Changelog
 ---------
 
+5.1.0
+.....
+
+Misc
+~~~~
```

Comment:
please remove. we don't know yet if we release 5.1 or 6 :)
the note itself is enough :)
```suggestion
```